metricbeat/module/vsphere/cluster/_meta/fields.yml

```diff
@@ -0,0 +1,38 @@
+- name: cluster
+  type: group
+  release: beta
+  description: >
+    Cluster information
+  fields:
+    - name: datastore.names
```

Comment:
Can we do nested mapping for all fields to consistency with other PRs?

    - name: datastore
      type: group
      fields:
          - name: names
            type: keyword
            description: >
              List of all the Datastore names associated with the cluster.
          - name: count
            type: long
            description: >
              Number of Datastores.